try {
  console.log( 'hi' + y )
} catch {
  console.log( 'you messed up' )
}